### PR TITLE
Add pow and selu operator test

### PR DESCRIPTION
**Pull request by houseroad**
Pow is not able to run on onnx-caffe2.

Selu is OK.

### CODE REVIEW

File: test/test_operators.py [hunk 1]
```diff
@@ -263,6 +263,15 @@ def test_logsoftmax(self):
...
         self.assertONNX(nn.LogSoftmax(dim=2), x)
 
+    def test_pow(self):
+        x = Variable(torch.randn(1, 2, 3, 4), requires_grad=True)
+        y = Variable(torch.randn(1, 2, 3, 4), requires_grad=True)
+        self.assertONNX(lambda x, y: x.pow(y), (x, y))
+
+    def test_selu(self):
+        x = Variable(torch.randn(1, 2, 3, 4), requires_grad=True)
+        self.assertONNX(nn.ELU(), x)
```
Comment (bddppq) [inline test/test_operators.py]: Should this be SELU instead of ELU?

**Comment by houseroad** [comment]
Good catch!

**Comment by bddppq** [comment]
What's the error you seen when running Pow in onnx-caffe2?

**Comment by houseroad** [comment]
In caffe2, Pow takes the exponent as an argument. But in ONNX, it is an input.
Probably not easy to solve it simply in onnx-caffe2. What do you think?

**Comment by bddppq** [comment]
Yeah in general it's impossible to use attribute to represent input (unless the input is actually static and our optimization passes can infer this fact), will need to extend caffe2 to take exponent as input.

**Comment by houseroad** [comment]
Shall I create a task? I think it's pretty fun for a bootcamper.

**Pull request merged**